app/controllers/api/users_controller.rb

```diff
@@ -3,6 +3,16 @@ module Api
   class UsersController < ApplicationController
     include Devise::Controllers::Helpers
 
+    def user_name
+      user = User.find_by(:id)
```

Comment:
This is not the syntax for `find_by`. Check [find_by](https://api.rubyonrails.org/v7.0.1/classes/ActiveRecord/FinderMethods.html#method-i-find_by) documentation.